content/en/apps/guides/debugging/Securely-sharing-your-development-environment.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "Securely sharing your development environment"
+linkTitle: "Securely sharing your development environment"
```

Comment:
The title is clearer now, although it should be in title case. 

Also, I would suggest making the filename more formal, since it will likely live on even if the title gets tweaks, eg `secure-sharing-of-developer-instance`

Reply:
Cool cool - should be fixed in 609baf2